### PR TITLE
[RDY] api/vim.c: Comment cleanup

### PR DESCRIPTION
**Pull request by ghost**
This should supersede #1481

### CODE REVIEW

File: src/nvim/api/vim.c [hunk 1]
```diff
@@ -568,7 +568,7 @@ Array vim_get_api_info(uint64_t channel_id)
 /// later.
 ///
 /// @param message The message to write
-/// @param to_err True if it should be treated as an error message(use
+/// @param to_err true if it should be treated as an error message(use
```
Comment (fwalch) [inline src/nvim/api/vim.c]: Space can be added before `(`

**Comment by fwalch** [comment]
The style guide says `[function] comments should be descriptive ("Opens the file") rather than imperative ("Open the file");`, so LGTM.

File: src/nvim/api/vim.c (hunk 1 above)
Comment (Pyrohh) [inline src/nvim/api/vim.c]: :+1:

**Comment by elmart** [comment]
That's a particular aspect of the style guide I don't agree with. I think imperative style is very often clearer and shorter. But if there's no consensus on changing that, I'm ok with it.

**Comment by Pyrohh** [comment]
I'm partial to the style guide's recommendation in this case; [this post](http://lua-users.org/lists/lua-l/2012-03/msg00785.html) shows some possible reasons for following it. Regardless, the main reason for this PR is consistency.

**Comment by Pyrohh** [comment]
Rebased and marking and as RDY if noone objects.

**Comment by Pyrohh** [comment]
@justinmk do you mind if I pull your changes from https://github.com/justinmk/neovim/commit/7b004c418568e34e00f7eb52c162d9cfdcd48944? I can just rename the PR to `Misc. comment improvements`.

**Comment by justinmk** [comment]
@Pyrohh sure.

Regarding this PR, in general I agree we should follow the style guide recommendation, for the reasons mentioned in your link. 

Sometimes imperative voice gains significant brevity and concision, then exceptions are allowed.

**Comment by Pyrohh** [comment]
Repushed with the mentioned changes.

> Sometimes imperative voice gains significant brevity and concision, then exceptions are allowed.

Indeed, although most if not all of these only needed a change of suffix.

**Comment by justinmk** [comment]
@Pyrohh next time please cherry-pick so author is preserved and reviewers can ignore commits they already reviewed.

**Comment by Pyrohh** [comment]
Okay, I can do that now if needed.

**Comment by justinmk** [comment]
No worries, I just need to verify something in my changes before committing this.

**Comment by Pyrohh** [comment]
Any comments @justinmk? This is ready on my end.

File: src/nvim/api/vim.c [hunk 2]
```diff
@@ -133,7 +133,7 @@ String vim_command_output(String str, Error *err)
   return cstr_to_string((char *)get_vim_var_str(VV_COMMAND_OUTPUT));
 }
 
-/// Evaluates the expression str using the vim internal expression
+/// Evaluates the expression str using vim's internal expression
```
Comment (justinmk) [inline src/nvim/api/vim.c]: old wording was better. Maybe capitalize `Vim`. Avoid possessive case in general.

File: src/nvim/api/vim.c [hunk 3]
```diff
@@ -218,7 +218,7 @@ ArrayOf(String) vim_list_runtime_paths(void)
   return rv;
 }
 
-/// Changes vim working directory
+/// Changes vim's working directory
```
Comment (justinmk) [inline src/nvim/api/vim.c]: old wording was better. Maybe capitalize `Vim`. Avoid possessive case in general.

File: src/nvim/api/vim.c (hunk 2 above)
Comment (Pyrohh) [inline src/nvim/api/vim.c]: Maybe `using the internal...`? I'll just use your suggestion otherwise.

Comment (justinmk) [inline src/nvim/api/vim.c]: `the vim internal expression` 

remember that neovim is vim, and for purposes of technical documentation, vim concepts are still vim concepts. We don't need to take ownership of vim concepts. nvim is vim.

Comment (Pyrohh) [inline src/nvim/api/vim.c]: Got it, repushed.

**Comment by justinmk** [comment]
merged, thanks

**Comment by Pyrohh** [comment]
Thanks.

**Pull request closed without merging**